src/Microsoft.Management.Configuration.Processor/Helpers/ConfigurationUnitInternal.cs

```diff
@@ -6,28 +6,38 @@
 
 namespace Microsoft.Management.Configuration.Processor.Helpers
 {
+    using System;
     using System.Collections.Generic;
+    using System.IO;
     using Microsoft.Management.Configuration.Processor.Constants;
+    using Microsoft.Management.Configuration.Processor.Exceptions;
     using Microsoft.PowerShell.Commands;
+    using Windows.Foundation.Collections;
 
     /// <summary>
     /// Wrapper around Configuration units and its directives. Creates a normalized directives map
     /// for consumption.
     /// </summary>
     internal class ConfigurationUnitInternal
     {
+        private static string configRootVar = "$WinGetConfigRoot";

```

Comment:
After thinking about this some, I don't think we should go with this form when we are doing a simple find and replace.  Without considering an actual syntax, we would convert `$WinGetConfigRootName => C:Name` if the file was `C:\config.yml`.

I think we should use one of the other forms of PowerShell syntax here to force it to be compliant (or at least, much more likely to be) while still being able to use a find and replace for time.  So either `${WinGetConfigRoot}` or `$($WinGetConfigRoot)`.